### ISSUE TITLE
Change the fluid value per ingot to 144mb

### ISSUE THREAD

**Issue by megaloloful** [EXTER7/Foundry#48]
I've realised that foundry uses a fluid value of 108mb/ingot unlike the 144mb used by every other mod with molten ingots. 
Also your fluid aren't properly fluid registered, (which is a good thing currently since it fixes a dupe with mods like TiCon or NeoTech since they use 144mb, but that isn't a long time solution in my opinion). 

I would highly suggest changing your value to 144mb and registering the fluids properly for better cross-mod compatibility and it also would safe you a lot of hassle since it looks like you currently re-add every tiCon casting table recipe with your  fluids and fluid values.

**Comment by Mowmaster** [comment]
Yes it is annoying to have foundry set fluids at 108mb where as tcon has 144mb the common factor is 432mb at a ratio of tcon:foundry of 4:3. its really annoying when you set enderpearls to melt down at 250mb (4 to make a bucket) and yet when you go to alloy it uses it at the 108mb rate. makes balancing very annoying...

I understand being different is cool. IC2 does a good job of that. but yeah, if you added a check to see if tcon is installed to use the 144mb would be awesome and make some of our lives easier. or even a config to set the value, that would be just about as good.
